requirements/_requirements.txt

```diff
@@ -23,6 +23,6 @@ pydantic~=2.6
 openai>=1.12.0
 structlog>=24.1.0
 zxing-cpp>=2.2.0
-boto3<=1.28.23
```

Comment:
could we align with current max version such that we do not move so far into the unknown versions

Reply:
Sure! I limited to current version!

Now that we understand the problem, we just have to remember to sometimes bump this limit. Or we could limit botocore and awscli versions too, because they're implicitly limited anyway.

Reply:
I am fine with both, yet prefer explicit